ddtrace_asgi/middleware.py

```diff
@@ -13,7 +14,8 @@ class TraceMiddleware:
     def __init__(
         self,
         app: ASGIApp,
-        tracer: Tracer,
+        *,
+        tracer: Tracer = tracer,
```

Comment:
Always been a bit cautious about defaults that refer to object… Can we turn this into `None` and then use the `if tracer is None: tracer = ddtrace.tracer` idiom?

Reply:
sure, like so?